concepts/tasks/about.md

```diff
@@ -0,0 +1,115 @@
+# About
+
+Tasks are processes meant to execute one specific operation.
+They usually don't communicate with other processes, but they can return a result to the process that started the task.
+
+Tasks are commonly used to parallelize work.
+
+## `async`/`await`
+
+To start a task, use [`Task.async/1`][task-async]. It takes an anonymous function as an argument and executes it in a new process that is linked to the caller process. It returns a [`%Task{}` struct][task-struct].
+
+To get the result of the execution, pass the `%Task{}` struct to [`Task.await/2`][task-await]. It will wait for the task to finish and return its result. The second argument is a timeout in milliseconds, defaulting to 5000. The value `:infinity` can be passed as a timeout to allow the task to execute without a timeout.
+
+```elixir
+task = Task.async(fn -> Enum.sum(0..999_999) end)
+# => %Task{
+#      owner: #PID<0.110.0>,
+#      pid: #PID<0.119.0>,
+#      ref: #Reference<0.3715954046.815529991.83334>
+#    }
+
+Task.await(task)
+# => 499999500000
+```
+
+Note that between starting the task and awaiting the task, the process that started the task is not blocked and might do other operations.
+
+Any task started with `Task.async/1` should be awaited because it will send a message to the calling process. `Task.await/2` can be called for each task only once.
+
+## `start`/`start_link`
+
+If you want to start a task for side-effects only, use [`Task.start/1`][task-start] or [`Task.start_link/1`][task-start-link]. `Task.start/1` will start a task that is not linked to the calling process, and `Task.start_link/1` will start a task that is linked to the calling process. Both functions return a `{:ok, pid}` tuple.
+
+## Timeouts
```

Comment:
I'm going to take this whole section and post it as a blog post. If you think this is too long for a `concept/about.md`, then I can shorten it and link to the post instead.

Reply:
I don't think it's too long. It's a complicated topic.